consensus/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "aleph-bft"
-version = "0.10.0"
+version = "0.10.1"
```

Comment:
I think bumping the version on anything that changes the external interface is a must, and we should probably also bump it when the final version is implemented.

Anyway, changed to "0.11.0", since you are right about the feature flag.